treeTrimming.js

```diff
@@ -46,6 +46,31 @@ function trimSilentTerminals(inputTree){
 	return trimSilentInner(treeCopy);
 }
 
+/* function removes non-lexical heads from a tree. Basically identical to trimSilentTerminals
```

Comment:
Am I right in thinking that markMinMax() doesn't ultimately use this? Do we actually need this function after all? Sorry, I'm sure we talked about this when you started on this issue, but I don't remember the details.

Reply:
In fact, isn't this function redundant with the more general trimAttributedNodes()?

Reply:
This is not used anywhere but Ed says it should work.